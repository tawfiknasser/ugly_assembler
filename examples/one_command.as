add r4,r7
