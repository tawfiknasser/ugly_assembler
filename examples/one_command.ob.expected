              2  0  
100       .....././///..
101       .../.....///..

